openedx/core/djangoapps/course_roles/models.py

```diff
@@ -0,0 +1,102 @@
+"""
+Models for course roles schema
+"""
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from organizations.models import Organization
+
+User = get_user_model()
+
+
+class CourseRolesRole(models.Model):
+    """
+    Model for a course roles role.
+
+    A role is a collection of permissions that can be assigned to a user.
```

Comment:
Should the comment include information about the name being stored in the code? This may be something we'd prefer to add once that portion of the code is written, but we should include that information at some point.

Reply:
I agree, I will keep that in mind when we add that part of the code, maybe it will be easier to reference at that time.